README.md

```diff
@@ -4,6 +4,25 @@ It basically only have tag feature, release subscription
 (but actually just role giving) feature.  
 I uploaded this for the demonstration of slash command.
 
+## Paginator
+This simple example shows how to easily create interactive, multiple page embeds that annyone can interact with.
+```py
+import discord
+import discord_slash
+from discord_slash import SlashCommand, SlashContext
+from discord_slash.utils import Paginator
+
+@slash.slash(name="command")
+async def command(ctx:SlashContext)
+embed1=discord.Embed(title="title")
+embed2=discord.Embed(title="another title")
+embed3=discord.Embed(title="yet another title")
+pages=[embed1,embed2,embed3]
+
+await Paginator(bot=self.bot,ctx=ctx,pages=pages,content="Hello there",prevLabel="Back",nextLabel="Forward",
+        prevEmoji="♥",nextEmoji="♥",prevStyle=1,nextStyle=2,indexStyle=3,timeout=10)
```

Comment:
This can appear a little confusing for documented purposes, also it's formatted oddly compared to usual PEP-8 styles.